butterfly-slack/src/main/resources/postAbort.json

```diff
@@ -0,0 +1,22 @@
+{
+  "channel": "%s",
+  "text": "Your Butterfly transformation failed",
+  "attachments": [
+    {
+      "text": "Error message goes here",
```

Comment:
Interpolate this with the actual error message (I know you already plan to do it, but I am just commenting so we keep track of this pending change)